jina/docker/database.py

```diff
@@ -49,21 +50,21 @@ def database(self):
     def collection(self):
         return self.database[self.collection_name]
     
-    def find(self, query: dict):
+    def find(self, query: Dict) -> None:
```

Comment:
Is there a way to know the key and value types of this Dictionary?

Reply:
Yes, its Dict[str, Union[Dict, List]]. Have made the addition.